x-pack/plugin/otel-data/src/main/java/org/elasticsearch/xpack/oteldata/OTelPlugin.java

```diff
@@ -33,9 +33,7 @@ public class OTelPlugin extends Plugin implements ActionPlugin {
     // This setting will be ignored if the plugin is disabled.
     static final Setting<Boolean> OTEL_DATA_REGISTRY_ENABLED = Setting.boolSetting(
         "xpack.otel_data.registry.enabled",
-        // OTel-data is under development, and we start with opt-in first.
-        // Furthermore, this could help with staged rollout in serverless
```

Comment:
By setting default to `true`, we kind of loose this. Is this relevant?I think we could still do stage rollout by setting it to `false` in serverless and gradually removing it.

Reply:
I might be missing something but I don't see much advantage of staged rollout. IIUC, the OTEL-data plugin will be engaged only when the OTEL mode will be used with the es-exporter which kind of gates the usage anyway.

Reply:
Yeah I think that may have come from me, but in retrospect it's not necessary. It was important for apm-data because of the impact on a GA product, but that's not the case here.